lingua_franca/internal.py

```diff
@@ -10,13 +10,13 @@
 
 
 _SUPPORTED_LANGUAGES = ("az", "ca", "cs", "da", "de", "en", "es", "fr", "hu",
-                        "it", "nl", "pl", "pt", "ru", "sl", "sv", "fa", "eu")
+                        "it", "nl", "pl", "pt", "ru", "sl", "sv", "fa", "eu", "uk")
 
 _SUPPORTED_FULL_LOCALIZATIONS = ("az-az", "ca-es", "cs-cz", "da-dk", "de-de",
                                  "en-au", "en-us", "es-es", "fr-fr",
                                  "hu-hu", "it-it", "nl-nl", "pl-pl",
                                  "fa-ir", "pt-pt", "ru-ru", "sl-si",
-                                 "sv-se", "tr-tr", "eu-eu")
+                                 "sv-se", "tr-tr", "eu-eu", "uk-uk")
```

Comment:
Lang code is uk-ua
See [here](http://www.lingoes.net/en/translator/langcode.htm)